### DISCUSSION TITLE
Shortened link used on localhost project doesn't redirect

### DISCUSSION THREAD

**Discussion by jhonellamaceren** [ash-jc-allen/short-url#108] [Q&A] [answered]
Hi, I have a localhost Laravel Project whose link is http://companyname.test

I initially added ` url('job/'.$pendingjob->slug)` as `destinationURL` argument, but the shortened URL keeps saving as
```
http://localhost/short/Key
```
which leads to a `404 not found` when I try it on my machine.

I tried to put the `http://companyname.test/job/'.$pendingjob->slug` and `http://127.0.0.1/job/'.$pendingjob->slug` directly to `destinationURL` , but they all lead to the same thing.

Is there a way so that these short URLs can actually redirect to my local project? 

Note: Fairly new to all this. Thank you in advanced

**Comment by ash-jc-allen** [comment, marked as answer]
Hey @jhonellamaceren! 😄

Apologies for taking a while to get back to you.

To get a better idea of the bug, could you maybe show me the code that you're using to create the Short URL? It'd also be really handy to see what row is being created in the database when you make it.
